Can_Node_Handler.cpp

```diff
@@ -75,6 +75,10 @@ void Can_Node_Handler::handleMessage(Frame& message) {
     writeThrottleMessages(0);
     return;
   }
+  
+  int16_t raw_RPM = Store().readMotorRpm(0);
+  int actual_motor_RPM = raw_RPM/4000; 
+  int wheel_RPM = actual_motor_RPM*2.17;//using gear ratio to convert Motor RPM to Wheel RPM
```

Comment:
You're storing the result of a float multiplied by an int as an int - is this what you want?